### ISSUE TITLE
advance_file_position_to_next_profile issue

### ISSUE THREAD

**Issue by rowanwins** [IQuOD/wodpy#6]
Gday,

Just wanted to say thanks for what appears to be a very promising python module!

The one thing I'm currently having issues with is working out how to move to the next profile in a file. Below is a simple example of what I'm trying. 

```
from wodpy import wod

fid = open("example.dat")
profile = wod.WodProfile(fid)

print profile.uid()

profile.advance_file_position_to_next_profile(fid)

print profile.uid()
```

returns

```
17361442
17361442
```

I'm sure I'm probably missing something really basic... :) 

Thanks,
Rowan

**Comment by katieannemills** [comment]
Nope, not your fault, our example there is just really terrible; do you get what you expect if you do:

```
from wodpy import wod

fid = open("example.dat")
profile = wod.WodProfile(fid)

print profile.uid()

profile = wod.WodProfile(fid)

print profile.uid()
```

I'll update the docs to try and avoid this misleading example in future - thanks, and let me know any other problems.

**Comment by rowanwins** [comment]
Thanks for the quick response @BillMills 
hmmm running with that I get 

```
KeyError: 'Significant digits'
```

I did run into some installation errors around numpy & pandas, I think due to me being on a windows machine, so I might double check that those dependencies are all functioning correctly, do you know if those dependencies require any specific versions?

**Comment by s-good** [comment]
Hi @rowanwins, thanks for trying out wodpy and for the feedback. The example.dat file only contains one profile, which is why you get an error message when trying to read a second one. Sorry for the confusion and for the very un-informative error message you encountered. I have created a pull request (#7) that means that a better error message will now be produced in this case.

You shouldn't need any specific versions of numpy or pandas.

**Comment by rowanwins** [comment]
Gday @s-good and @BillMills 

Ok so I've out what the problem is partly...

I was using a file I downloaded from the WODselect site however it looks like the file was not well structured. Using the csvFromWOD.exe program I was able to generate a csv however it seemed like it's structure was somewhat off. When I retried things on a different WOD Dataset (one of the precanned geographic regions ones) things worked (including wodpy) as expected.

So I think I need to have a play with how to extract data from WOD in the most suitable format, if either of you has any advice as to what you normally do that would be great!

Thanks,
Rowan
